### PR TITLE
added float resolution

### PR DESCRIPTION
**Pull request by SucyLi**

### CODE REVIEW

File: CPAC/nuisance/nuisance.py [hunk 1]
```diff
@@ -1078,7 +1078,7 @@ def create_nuisance_workflow(nuisance_selectors,
                     )
                     
                     pc_node = pe.Node(
-                        PC(args='-vmean -nscale', pcs=regressor_selector['summary']['components']),
+                        PC(args='-vmean -nscale', pcs=regressor_selector['summary']['components'], outputtype='NIFTI'),
```
Comment (anibalsolon) [inline CPAC/nuisance/nuisance.py]: Testing comment

File: CPAC/pipeline/cpac_pipeline.py [hunk 2]
```diff
@@ -3824,7 +3900,7 @@ def prep_workflow(sub_dict, c, run, pipeline_timing_info=None,
 
                 
             # Actually run the pipeline now, for the current subject
-            workflow.run(plugin=plugin, plugin_args=plugin_args)
+            workflow.run(plugin='Linear', plugin_args=plugin_args)
```
Comment (sgiavasis) [inline CPAC/pipeline/cpac_pipeline.py]: Just need to change this back.

Comment (SucyLi) [reply CPAC/pipeline/cpac_pipeline.py]: changed back

File: CPAC/resources/configs/pipeline_config_template.yml [hunk 3]
```diff
@@ -409,19 +409,22 @@ runRegisterFuncToMNI :  [1]
 
 
 # The resolution (in mm) to which the preprocessed, registered functional timeseries outputs are written into. Note that selecting a 1 mm or 2 mm resolution might substantially increase your RAM needs- these resolutions should be selected with caution. For most cases, 3 mm or 4 mm resolutions are suggested.
-resolution_for_func_preproc :  3mm
+resolution_for_func_preproc :  3 # if 3 dim different, enter: num1xnum2xnum3, eg. 3x2.5x2.5
```
Comment (sgiavasis) [inline CPAC/resources/configs/pipeline_config_template.yml]: Can we put this comment in the line above, as an example?

Comment (SucyLi) [reply CPAC/resources/configs/pipeline_config_template.yml]: oh sure. have changed that

**Pull request merged**